content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/image.html

```diff
@@ -26,11 +29,15 @@
        class="cmp-image__link"
        data-sly-attribute="${image.imageLink.htmlAttributes}"
        data-cmp-clickable="${image.data ? true : false}">
-        <img srcset="${image.srcset}" src="${image.src}"
+        <img src="${image.src}"
+             srcset="${image.srcset}"
              loading="${image.lazyEnabled ? 'lazy' : ''}"
              class="cmp-image__image"
              itemprop="contentUrl"
+             data-sizes="${image.autoSizesEnabled ? 'auto' : ''}"
+             data-srcset="${image.srcset}"
```

Comment:
Why do we need `srcset` and `data-srcset` in the markup if they have the same initial value?